ts/features/itwallet/issuance/components/ItwCredentialPreviewClaimsList.tsx

```diff
@@ -0,0 +1,61 @@
+import { Divider, H6, IOStyles } from "@pagopa/io-app-design-system";
+import React from "react";
+import { View } from "react-native";
+import I18n from "../../../../i18n";
+import { ItwCredentialClaim } from "../../common/components/ItwCredentialClaim";
+import { ItwReleaserName } from "../../common/components/ItwReleaserName";
+import {
+  getCredentialStatus,
+  parseClaims,
+  WellKnownClaim
+} from "../../common/utils/itwClaimsUtils";
+import { StoredCredential } from "../../common/utils/itwTypesUtils";
+
+type ItwCredentialClaimsListProps = {
+  data: StoredCredential;
+};
+
+/**
+ * This component renders the list of claims for a credential.
+ * It dinamically renders the list of claims passed as claims prop in the order they are passed.
+ * @param data - the {@link StoredCredential} of the credential.
+ */
+const ItwCredentialPreviewClaimsList = ({
+  data
+}: ItwCredentialClaimsListProps) => {
+  const credentialStatus = getCredentialStatus(data);
+
+  const claims = parseClaims(data.parsedCredential, {
+    exclude: [WellKnownClaim.unique_id, WellKnownClaim.link_qr_code]
+  });
+
+  return (
+    <>
+      <View style={IOStyles.rowSpaceBetween}>
+        <H6 color="grey-700">
+          {I18n.t(
+            "features.itWallet.presentation.credentialDetails.documentDataTitle"
+          )}
+        </H6>
+      </View>
+      {claims.map((elem, index) => (
+        <View key={index}>
+          <ItwCredentialClaim
+            claim={elem}
+            isPreview={true}
+            hidden={false}
+            credentialStatus={credentialStatus}
```

Comment:
I think we can remove the credential status when `ItwCredentialClaim` is in preview mode. It is optional, so it's safe to omit it. Within `ItwCredentialClaim` it is used to show the validity badge in date claims.